File: src/main/java/com/su/model/Comment.java
package com.su.model;

import lombok.Data;
import lombok.NoArgsConstructor;

import java.util.Date;

@Data
@NoArgsConstructor
public class Comment {
    private long time;
    private String author;
    private String text;

    public Comment(String author, String text) {
        this.time = new Date().getTime();
        this.author = author;
        this.text = text;
    }
}

File: src/main/frontend/src/components/Comment.js
import React from 'react';
import {Panel} from 'react-bootstrap';
import Moment from 'react-moment';

class Comment extends React.Component {

    render() {
        return <Panel className="Comment">
            <div className="Comment__info">
                <div className="Comment__author">{this.props.author}</div>
                <div className="Comment__time">
                    <Moment date={new Date(this.props.time)} format="DD.MM.YYYY HH:mm"/>
                </div>
            </div>
            <div className="Comment__text">
                <span>{this.props.text}</span>
            </div>
        </Panel>;
    }
}

export default Comment;